### ISSUE TITLE
"You need to authorize Trello"

### ISSUE THREAD

**Issue by adrienjoly** [hewigovens/trello-this#1]
RIght after installing the chrome extension:
- I opened a new tab on fhacktory.com
- right clicked on the content of the page
- selected "card this to trello"
- got a "You need to authorize Trello" pop-in
  => what am I supposed to do at this point? (or how?)

**Comment by hewigovens** [comment]
Yeah It's kind of a bug, you can click anywhere except the popup then login
with your Trello account, then try click "Card this to trello" again, It
should be work.

On Wed, Jun 25, 2014 at 5:33 PM, Adrien Joly notifications@github.com
wrote:

> RIght after installing the chrome extension:
> - I opened a new tab on fhacktory.com
> - right clicked on the content of the page
> - selected "card this to trello"
> - got a "You need to authorize Trello" pop-in => what am I supposed to
>   do at this point? (or how?)
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/hewigovens/trello-this/issues/1.

## 

Sincerely,
hewig xu

**Comment by adrienjoly** [comment]
the first step worked, then i got that: https://www.dropbox.com/s/eiycoftdcmt898n/Screenshot%202014-06-26%2000.19.00.png

**Comment by hewigovens** [comment]
Sorry, I will fix that later, here is the workaround : you can re-select
text then click "Card this to Trello" again

On Thu, Jun 26, 2014 at 6:19 AM, Adrien Joly notifications@github.com
wrote:

> the first step worked, then i got that:
> https://www.dropbox.com/s/eiycoftdcmt898n/Screenshot%202014-06-26%2000.19.00.png
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/hewigovens/trello-this/issues/1#issuecomment-47166432
> .

## 

Sincerely,
hewig xu

**Comment by adrienjoly** [comment]
ok it seems to work, now. i guess it was taking some time for the extension
to load my boards from trello's api.

thanks!

On Thu, Jun 26, 2014 at 5:10 AM, hewigovens notifications@github.com
wrote:

> Sorry, I will fix that later, here is the workaround : you can re-select
> text then click "Card this to Trello" again
> 
> On Thu, Jun 26, 2014 at 6:19 AM, Adrien Joly notifications@github.com
> wrote:
> 
> > the first step worked, then i got that:
> > 
> > https://www.dropbox.com/s/eiycoftdcmt898n/Screenshot%202014-06-26%2000.19.00.png
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > <
> > https://github.com/hewigovens/trello-this/issues/1#issuecomment-47166432>
> > .
> 
> ## 
> 
> Sincerely,
> hewig xu
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/hewigovens/trello-this/issues/1#issuecomment-47183523
> .

## 

Adrien Joly
adrienjoly.com/
whyd.com/adrien

**Comment by tarcisiogf** [comment]
Up to now, I still get this message
